internal/web_server/web/views/pages/dashboard_pages/user_admin_page.templ

```diff
@@ -0,0 +1,19 @@
+package dashboard_pages
+
+import (
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/layout_components"
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/dashboard_components/user_admin"
+	"github.com/Molnes/Nyhetsjeger/internal/models/users/usernames"
+)
+
+
+// The username administration page main function
+templ UsernameAdminPage(data *usernames.UsernameAdminInfo) {
+	@layout_components.DashBoardLayout("Toppliste") {
+		<div class="flex flex-col items-center gap-6 max-w-screen-sm m-auto p-5">
+			<h2 class="text-xl font-bold">Brukernavn administrasjon</h2>
+			<input class="border border-gray-600 rounded-input px-2" type="text" placeholder="søk her"/>
```

Comment:
tried running it, this search does not seem to be implemented?

Reply:
i will do this in 3 stages to avoid too big merge requests.
1. Tables
2. Editing usernames
3. Search functionality if even needed